cmd/gitops/install/cmd.go

```diff
@@ -84,7 +85,25 @@ func installRunCmd(cmd *cobra.Command, args []string) error {
 	}
 
 	if installParams.DryRun {
-		manifests, err := models.BootstrapManifests(fluxClient, clusterName, namespace, configURL)
+		fluxNamespace, err := kubeClient.FetchNamespaceWithLabel(ctx, flux.PartOfLabelKey, flux.PartOfLabelValue)
+		if err != nil {
+			if !errors.Is(err, kube.ErrNamespaceNotFound) {
+				return fmt.Errorf("failed fetching flux namespace: %w", err)
+			}
+		}
+
+		if fluxNamespace == nil {
+			fluxNamespace = &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{
```

Comment:
I wonder if we really need this whole object. I only see we are using `fluxNamespace.Name` after this line 🤔